ReleaseProcessScript/tools/Core/main_helper_functions.ps1

```diff
@@ -49,6 +49,36 @@ function Get-Develop-Current-Version ($StartReleasebranch)
     return Read-Current-Version
   }
 
+  $MostRecentVersion = Get-Develop-Most-Recent-Version
+    
+  $PossibleVersions = Get-Possible-Next-Versions-Develop $MostRecentVersion $WithoutPrerelease
+  Write-Host "Please choose Release Version:"
```

Comment:
where did we have this message before? And why does it remain there?

Reply:
Before, the message "Please choose Release Version:" was only printed in the function `Get-Develop-Current-Version` (and not `Get-Hotfix-Current-Version`). Now, this message is printed between the assignments `$PossibleVersions = ...` and `$CurrentVersion = ...` in both cases.

I assumed that was an oversight, which is why I added the `Write-Host ...` - or was it intentional?

Reply:
Ah there it is. Sometimes, the diff-viewer isn't very helpful...